### PR TITLE
Add workflow for win

### PR DESCRIPTION
**Pull request by 1e-to**
Added build of dpсtl package to github runner on windows
Tested locally run: https://github.com/1e-to/dpctl/pull/4

### CODE REVIEW

File: .github/workflows/conda-package.yml [hunk 1]
```diff
@@ -8,50 +8,53 @@ env:
...
+        os: [ubuntu-latest, windows-latest]
+        include:
+        - os: ubuntu-latest
+          pkgs_dirs: ~/.conda/pkgs
+          condarc: ~/.condarc
+          conda-bld: /usr/share/miniconda/conda-bld/linux-64/
+        - os: windows-latest
+          pkgs_dirs: C:\Users\runneradmin\.conda\pkgs
+          condarc: C:\Users\runneradmin\.condarc
+          conda-bld: C:\Miniconda\conda-bld\
```
Comment (oleksandr-pavlyk) [inline .github/workflows/conda-package.yml:23]: Shouldn't this be `C:\Miniconda\conda-bdl\win-64`?

File: .github/workflows/conda-package.yml [hunk 2]
```diff
@@ -8,50 +8,53 @@ env:
...
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
+        include:
+        - os: ubuntu-latest
+          pkgs_dirs: ~/.conda/pkgs
+          condarc: ~/.condarc
+          conda-bld: /usr/share/miniconda/conda-bld/linux-64/
+        - os: windows-latest
+          pkgs_dirs: C:\Users\runneradmin\.conda\pkgs
```
Comment (oleksandr-pavlyk) [inline .github/workflows/conda-package.yml:21]: Can we use Windows env variables like `%HOMEDRIVE%%HOMEPATH%` to replace hard-coded home path which may change down the road?

Comment (1e-to) [reply .github/workflows/conda-package.yml:21]: There is currently no way to use environment variables here

File: .github/workflows/conda-package.yml [hunk 3]
```diff
@@ -8,50 +8,53 @@ env:
...
-            $VERSIONS \
-            $CHANNELS \
-            conda-recipe
+        run: conda build --no-test --python ${{ matrix.python }} -c intel -c defaults --override-channels conda-recipe
       - name: Upload artifact
         uses: actions/upload-artifact@v2
         with:
-          name: ${{ env.PACKAGE_NAME }} ${{ runner.os }} Python ${{ matrix.python }}
-          path: /usr/share/miniconda/conda-bld/linux-64/${{ env.PACKAGE_NAME }}-*.tar.bz2
+          name: ${{ env.PACKAGE_NAME }} ${{ matrix.os }} Python ${{ matrix.python }}
```
Comment (PokhodenkoSA) [inline .github/workflows/conda-package.yml:56]: matrix.os = ubuntu-latest
runner.os = Linux

Test job uses runner.os for searching for artifacts.

**Review by PokhodenkoSA — CHANGES_REQUESTED** [review]

**Comment by oleksandr-pavlyk** [comment]
@1e-to Is this PR still active? Why doesn't the workflow run?

**Comment by 1e-to** [comment]
> @1e-to Is this PR still active? Why doesn't the workflow run?

There were some problems with ocl drivers for windows, I solved them here https://github.com/1e-to/dpctl/pull/5
Now everything is settled and working

**Review by PokhodenkoSA — APPROVED** [review]

**Pull request merged**